components/stack/stack.test.js

```diff
@@ -0,0 +1,168 @@
+import { assert } from 'chai';
+import { createLocalVue, shallowMount } from '@vue/test-utils';
+import { DT_STACK_DIR, DT_STACK_GAP } from './stack_constants';
+import DtStack from './stack.vue';
+
+// Constants
+const baseSlotsData = {
+  default: `<div data-qa="default-content">Test 1</div> <div>Test 2</div>`,
+};
+
+describe('DtStack Tests', function () {
+  // Wrappers
+  let wrapper;
+
+  let defaultContent;
+
+  // Environment
+  let slots = baseSlotsData;
+
+  // Helpers
+  const _setChildWrappers = () => {
+    defaultContent = wrapper.find('[data-qa="default-content"]');
+  };
+
+  const _setWrappers = () => {
+    wrapper = shallowMount(DtStack, {
+      slots,
+      localVue: this.localVue,
+    });
+    _setChildWrappers();
+  };
+
+  // Setup
+  before(function () {
+    this.localVue = createLocalVue();
+  });
+
+  // Teardown
+  afterEach(function () {
+    slots = baseSlotsData;
+  });
+
+  beforeEach(function () { _setWrappers(); });
+
+  describe('Presentation Tests', function () {
```

Comment:
Nice tests, good pattern of setting state in describe/beforeEach and then asserting within the "it"